vllm/model_executor/models/gpt2.py

```diff
@@ -297,7 +300,7 @@ def sample(
     def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
         params_dict = dict(self.named_parameters(remove_duplicate=False))
         for name, loaded_weight in weights:
-            if "lm_head.weight" in name:
+            if "lm_head" in name:
```

Comment:
```suggestion
            if name.startswith("lm_head"):
```

I think this is more robust?